.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+branches:
+  only:
+    - master
+    - /\bv[0-9]/
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
+script:
+  - flake8 qatrfm
+  - flake8 tests
+deploy:
+  provider: pypi
+  user: "jlausuch"
+  password: $PYPI_PASS
```

Comment:
Comes PYPI_PASS from shell environment?
What I found so far is: https://docs.travis-ci.com/user/deployment/pypi/ which has some encryption mechanism

Reply:
thanks for the pointer, but will remove pypi for now.